robot.py

```diff
@@ -176,6 +171,18 @@ def teleopPeriodic(self) -> None:
         if self.codriver.getAButton():
             self.chassis.set_pose_failsafe()
 
+        # Climb
+        if self.codriver.getYButton():
```

Comment:
In the past we have used two simultaneous button presses to activate things like this - accidental activation during the game would be a disaster!
Can we get back to regular play if this controller is activated? Is it worth putting in that ability?